pom.xml

```diff
@@ -58,6 +58,21 @@
       <artifactId>structured-logging</artifactId>
       <version>${structured-logging.version}</version>
     </dependency>
+    <dependency>
+      <groupId>uk.gov.companieshouse</groupId>
+      <artifactId>api-sdk-manager-java-library</artifactId>
+      <version>1.0.4</version>
+    </dependency>
+    <dependency>
+      <groupId>uk.gov.companieshouse</groupId>
```

Comment:
Just wondering, if we're hitting the private API to get the e-mail, do we need to include this public SDK too? Quick check - without it, does the call still work?

Reply:
we had to add it for some imports, think it was some exceptions being thrown from private sdk